src/app/app.component.scss

```diff
@@ -1 +1,43 @@
 @import "main";
+
```

Comment:
I don't see a `main.scss` file anywhere. BUT, in a later comment, I ask you to add this file. See that comment.